grapple/types/interfaces.py

```diff
@@ -209,3 +209,18 @@ def resolve_raw_value(self, info, **kwargs):
             return self.value.source
 
         return self.value
+
+
+def get_snippet_interface():
+    return import_string(grapple_settings.SNIPPET_INTERFACE)
+
+
+class SnippetInterface(graphene.Interface):
+    snippet_type = graphene.String(required=True)
+
+    @classmethod
+    def resolve_type(cls, instance, info, **kwargs):
+        return registry.snippets[type(instance)]
+
+    def resolve_snippet_type(self, info, **kwargs):
+        return self.__class__.__name__
```

Comment:
Should we be using the `appname.ModelName` form? Does Wagtail require snippet names to be unique?

Reply:
Well, for the Page type, we have the `pageType` field, which returns the page class name, and `contentType`, which returns the `appname.ModelName` form. We could add the `contentType` field to `SnippetInterface` too.